crates/rbuilder/src/live_builder/block_output/bidding/interfaces.rs

```diff
@@ -51,7 +51,7 @@ impl Bid {
     }
 }
 
-/// Makes the actual bid (send it to the relay)
+/// Makes the actual bid (seal + send it to the relay).
 pub trait BidMaker: std::fmt::Debug {
```

Comment:
to be more rusty and help with the issue you've mentioned to me about finding it difficult to differentiate between trait and struct naming in rust, you could consider this:

```suggestion
/// Makes the actual bid (seal + send it to the relay).
pub trait MakeBid: std::fmt::Debug {
```

remember rust traits are best thought of as behaviours, they're not analogous to c++ interfaces. this file should be renamed to `traits.rs`.

Reply:
I'll think about this.